lib/Auth/AWS.php

```diff
@@ -212,7 +212,7 @@ private function hmacsha1(string $key, string $message): string
         }
         $key = str_pad($key, $blocksize, chr(0x00));
         $ipad = str_repeat(chr(0x36), $blocksize);
-        $opad = str_repeat(chr(0x5c), $blocksize);
+        $opad = str_repeat(chr(0x5C), $blocksize);
```

Comment:
The codecov complaint is related to issue #193 
I won't bother removing this code in the 5.1 release series, it can be removed in major version 6.